src/main.rs

```diff
@@ -367,7 +367,7 @@ async fn use_node(
             .await
             .context("Error querying account list.")?;
         // this relies on the fact that get_account_list returns canonical addresses.
-        log::debug!("Initializing the address cache with {} accounts.", accounts.len());
+        log::info!("Initializing the address cache with {} accounts.", accounts.len());
```

Comment:
This is more an implementation detail, so I am not sure why it warrants being `info`. It is mostly useful for debugging and not in production.

Reply:
I was thinking that it was fine to bump level as it's only emitted once per connection. But it's fine, I'll revert.